### PR TITLE
[Fix](cloud) Fix auto partition cannot load data

### PR DESCRIPTION
**Pull request by zclllyybb**
## Proposed changes

Issue Number: close #xxx

in cloud mode, run test_select_with_predicate_prune could reappear it.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zclllyybb** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/catalog/Tablet.java [hunk 1]
```diff
@@ -246,6 +247,34 @@ public Multimap<Long, Long> getNormalReplicaBackendPathMap() throws UserExceptio
         return map;
     }
 
+    // for cloud mode without ConnectContext. use BE IP to find replica
+    protected Multimap<Long, Long> getNormalReplicaBackendPathMap(String be_address) throws UserException {
```
Comment (luwei16) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/Tablet.java:251]: Should this function be implemented in cloud tablet?

Comment (zclllyybb) [reply fe/fe-core/src/main/java/org/apache/doris/catalog/Tablet.java:251]: yes, it did. there's call from CloudTablet to this

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
http://43.132.222.7:8111/test/-7872041104277188299?currentProjectId=Doris_DorisCloudRegression&expandTestHistoryChartSection=true
passed relative cloud test

File: gensrc/thrift/FrontendService.thrift [hunk 2]
```diff
@@ -1257,6 +1257,7 @@ struct TCreatePartitionRequest {
     3: optional i64 table_id
     // for each partition column's partition values. [missing_rows, partition_keys]->Left bound(for range) or Point(for list)
     4: optional list<list<Exprs.TStringLiteral>> partitionValues
+    5: optional string be_address
```
Comment (gavinchou) [inline gensrc/thrift/FrontendService.thrift:1260]: Add comment on what it consists of.
Consider naming "be_endpoint"

**Comment by zclllyybb** [comment]
run buildall

**Comment by zclllyybb** [comment]
run buildall

**Review by luwei16 — APPROVED** [review]

**Review by gavinchou — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**